web/src/App.tsx

```diff
@@ -1,21 +1,21 @@
-import {TourProvider} from '@reactour/tour';
 import * as Sentry from '@sentry/react';
 import './App.less';
 import AnalyticsProvider from './components/Analytics/AnalyticsProvider';
 import ErrorBoundary from './components/ErrorBoundary';
+import {TracetestTourProvider} from './components/GuidedTour/TracetestTourProvider';
```

Comment:
Super minor 😄 if the component name is `GuidedTour` I would prefer to name it `GuidedTourProvider`